unstructured/ingest/connector/astra.py

```diff
@@ -123,3 +130,117 @@ def normalize_dict(self, element_dict: dict) -> dict:
             "content": element_dict.pop("text", None),
             "metadata": element_dict,
         }
+
+
+@dataclass
+class AstraIngestDoc(IngestDocCleanupMixin, BaseSingleIngestDoc):
+    connector_config: SimpleAstraConfig
+    metadata: t.Dict[str, str] = field(default_factory=dict)
+    registry_name: str = "astra"
+
+    @property
+    def filename(self):
+        return (
+            Path(self.processor_config.output_dir)
```

Comment:
This is going to the wrong folder. It needs to go to the download folder so that the Partitioner can pick it up. So something like this:
```
    @property
    def filename(self):
        return (
            Path(self.read_config.download_dir)
            / self.connector_config.collection_name
            / f"{self.metadata['_id']}.txt"
        ).resolve()
```